tests/dummy/app/pods/date-time/controller.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember'
+const {
+  inject: {service}
```

Comment:
fyi, our agreed upon 'ciena' style here is to use up to only one level of destructuring for imports. so just up to the 'inject'.